Configuration/PyReleaseValidation/python/relval_premix.py

```diff
@@ -80,3 +80,18 @@
 workflows[250407] = ['NuGun_UP15',["FS_NuGun_UP15_PRMXUP15_PU25","HARVESTUP15FS","MINIAODMCUP15FS"]]
 workflows[250408] = ['QCD_FlatPt_15_3000HS_13',["FS_QCD_FlatPt_15_3000HS_13_PRMXUP15_PU25","HARVESTUP15FS","MINIAODMCUP15FS"]]
 workflows[250409] = ['SMS-T1tttt_mGl-1500_mLSP-100_13',["FS_SMS-T1tttt_mGl-1500_mLSP-100_13_PRMXUP15_PU25","HARVESTUP15FS","MINIAODMCUP15FS"]]
+
+#fastsim,  2017 PU50####
+
+## premixed minbias
+workflows[250399.17]=['',['FS_PREMIXUP17_PU50']]
+## signal + PU
+workflows[250400.17] = ['ZEE_13',["FS_ZEE_13_PRMXUP17_PU50","HARVESTUP17FS","MINIAODMCUP17FS"]]
+workflows[250402.17] = ['TTbar_13',["FS_TTbar_13_PRMXUP17_PU50","HARVESTUP17FS","MINIAODMCUP17FS"]]
+workflows[250402.117] = ['TTbar_13',["FS_PREMIXUP17_PU50","FS_TTbar_13_PRMXLOCALUP17_PU50","HARVESTUP17FS"]]
```

Comment:
Could you please use `.171` instead to follow the convention of similar FullSim workflows?

Reply:
good point. Thanks for your comment.